2022-06-14 06:39:57.725592: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libnvidia-fatbinaryloader.so.396.82: cannot open shared object file: No such file or directory
2022-06-14 06:39:57.725639: E tensorflow/stream_executor/cuda/cuda_driver.cc:313] failed call to cuInit: UNKNOWN ERROR (303)
2022-06-14 06:39:57.725668: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (kartavya): /proc/driver/nvidia/version does not exist
2022-06-14 06:39:58.045809: I tensorflow/core/platform/cpu_feature_guard.cc:143] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2022-06-14 06:39:58.053140: I tensorflow/core/platform/profile_utils/cpu_utils.cc:102] CPU Frequency: 2195000000 Hz
2022-06-14 06:39:58.053335: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x60b9660 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2022-06-14 06:39:58.053361: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
models/shakespeare/stacked_lstm.py
############################## shakespeare.stacked_lstm ##############################
Namespace(aggregation='mean', batch_size=8, clients_per_round=30, corruption='flip', data_dir='data', dataset='shakespeare', decay_lr_every=50, eval_every=20, fraction_corrupt=0.25, lr=0.64, lr_decay=2.0, minibatch=None, model='stacked_lstm', no_logging=False, num_epochs=1, num_rounds=600, output_summary_file='outputs/outputs_shakespeare_avg_flip_0.25_1', patience_iter=20, seed=1, validation=False, weiszfeld_maxiter=4)
Traceback (most recent call last):
  File "models/main.py", line 177, in <module>
    main()
  File "models/main.py", line 57, in main
    server_model = ServerModel(ClientModel(*model_params, seed=args.seed - 1))
  File "/home/joker/Desktop/Capstone/RFA_capstone/models/shakespeare/stacked_lstm.py", line 16, in __init__
    super(ClientModel, self).__init__(lr, seed, max_batch_size)
  File "/home/joker/Desktop/Capstone/RFA_capstone/models/model.py", line 37, in __init__
    self.sess.run(tf.compat.v1.global_variables_initializer())
  File "/home/joker/.local/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 958, in run
    run_metadata_ptr)
  File "/home/joker/.local/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1181, in _run
    feed_dict_tensor, options, run_metadata)
  File "/home/joker/.local/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1359, in _do_run
    run_metadata)
  File "/home/joker/.local/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1365, in _do_call
    return fn(*args)
  File "/home/joker/.local/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1350, in _run_fn
    target_list, run_metadata)
  File "/home/joker/.local/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1443, in _call_tf_sessionrun
    run_metadata)
KeyboardInterrupt
